### PR TITLE
 Implementing FailedToLoadView

### PR DESCRIPTION
**Pull request by AnnaBanani**
Xib and Swift files FailedToLoadView created
The button "Generate new artwork" to FeaturedArtWorkLoading View added

### CODE REVIEW

File: TheMET/TheMET/FailedToLoadView/FailedToLoadView.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        textLabel.textColor = UIColor(red: 188.0/255, green: 176.0/255, blue: 193.0/255, alpha: 1)
+        imageView.image = UIImage(named: "No Connection")
+        button.backgroundColor = UIColor(red: 134.0/255, green: 121.0/255, blue: 139.0/255, alpha: 1)
+        button.layer.cornerRadius = 30
+        var buttonAttributed: [NSAttributedString.Key: Any] = [
+            NSAttributedString.Key.foregroundColor: UIColor(red: 242.0/255, green: 242.0/255, blue: 242.0/255, alpha: 1)
+        ]
+        let buttonTitle: String = "Try again"
+        let attributedNexttButtonString = NSAttributedString(string: buttonTitle, attributes: buttonAttributed)
+        button.setAttributedTitle(attributedNexttButtonString, for: .normal)
```
Comment (Ssimboss) [inline TheMET/TheMET/FailedToLoadView/FailedToLoadView.swift:36]: I guess such simple task like setting up the text and colour of the text on button can be resolved by simple
[setTitleColor](https://developer.apple.com/documentation/uikit/uibutton/1623993-settitlecolor) and [setTitle](https://developer.apple.com/documentation/uikit/uibutton/1624018-settitle) methods. No need for the `NSAttributedString`.

File: TheMET/TheMET/FeaturedArtWorkLoadingView/FeaturedArtWorkLoadingView.swift [hunk 2]
```diff
@@ -26,9 +26,14 @@ class FeaturedArtWorkLoadingView: UIView {
...
         textLabel.textColor = UIColor(red: 188.0/255, green: 176.0/255, blue: 193.0/255, alpha: 1)
         imageView.image = UIImage(named: "Frame")
+        button.backgroundColor = UIColor(red: 134.0/255, green: 121.0/255, blue: 139.0/255, alpha: 1)
+        button.layer.cornerRadius = 30
+        var buttonAttributed: [NSAttributedString.Key: Any] = [
+            NSAttributedString.Key.foregroundColor: UIColor(red: 242.0/255, green: 242.0/255, blue: 242.0/255, alpha: 1)
+        ]
+        let buttonTitle: String = "Generate new artwork"
+        let attributedNexttButtonString = NSAttributedString(string: buttonTitle, attributes: buttonAttributed)
+        button.setAttributedTitle(attributedNexttButtonString, for: .normal)
```
Comment (Ssimboss) [inline TheMET/TheMET/FeaturedArtWorkLoadingView/FeaturedArtWorkLoadingView.swift:36]: The same issue with overcomplicated solution

File: TheMET/TheMET/FailedToLoadView/FailedToLoadView.xib [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            </subviews>
+            <viewLayoutGuide key="safeArea" id="vUN-kp-3ea"/>
+            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+            <constraints>
+                <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="bbX-wp-Twx" secondAttribute="trailing" constant="50" id="0Px-Y4-Fph"/>
+                <constraint firstItem="ebe-23-fjT" firstAttribute="top" secondItem="vUN-kp-3ea" secondAttribute="top" constant="120" id="Hta-8z-i5m"/>
+                <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="zpv-wu-J7I" secondAttribute="trailing" constant="50" id="NBs-nV-3Wf"/>
+                <constraint firstItem="bbX-wp-Twx" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="50" id="UZT-5i-IUg"/>
+                <constraint firstItem="zpv-wu-J7I" firstAttribute="centerX" secondItem="vUN-kp-3ea" secondAttribute="centerX" id="agr-fH-4dq"/>
+                <constraint firstItem="bbX-wp-Twx" firstAttribute="centerX" secondItem="vUN-kp-3ea" secondAttribute="centerX" id="ajw-Ep-coN"/>
```
Comment (Ssimboss) [inline TheMET/TheMET/FailedToLoadView/FailedToLoadView.xib:46]: Here and in other XIB:
There is no need to center button and title horizontally. They have equal leading and trailing insets. It means they are already centered

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/FeaturedArtWorkLoadingView/FeaturedArtWorkLoadingView.swift (hunk 2 above)
Comment (AnnaBanani) [reply TheMET/TheMET/FeaturedArtWorkLoadingView/FeaturedArtWorkLoadingView.swift:36]: fixed

File: TheMET/TheMET/FailedToLoadView/FailedToLoadView.xib (hunk 3 above)
Comment (AnnaBanani) [reply TheMET/TheMET/FailedToLoadView/FailedToLoadView.xib]: fixed

**Review by Ssimboss — APPROVED** [review]

**Pull request merged**